runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -112,4 +122,68 @@ public void onDecorativeObjectDespawned(DecorativeObjectDespawned event)
 		DecorativeObject object = event.getDecorativeObject();
 		streams.remove(object);
 	}
+
+	//Shows the player's lowest experience skill
+	@Subscribe
+	public void onGameTick(GameTick gameTick)
+	{
+		if (client.getGameState() != GameState.LOGGED_IN)
+		{
+			return;
+		}
+		if (client.getLocalPlayer().getWorldLocation().getRegionID() == TOG_REGION)
+		{
+			activateTearsOverlay();

```

Comment:
what is the point of having separate methods that mostly do nothing for this? just write the logic here